src/vcpkg/commands.integrate.cpp

```diff
@@ -396,14 +387,14 @@ CMake projects should use: "-DCMAKE_TOOLCHAIN_FILE=%s"
 #endif
 
         was_deleted |= fs.remove(get_path_txt_path(), VCPKG_LINE_INFO);
-
+        auto message = msg::format(msgUserWideIntegration);
         if (was_deleted)
         {
-            print2(Color::success, "User-wide integration was removed\n");
+            msg::println(message.append_raw("was removed."));
         }
         else
         {
-            print2(Color::success, "User-wide integration is not installed\n");
+            msg::println(message.append_raw("is not installed."));
```

Comment:
These need to be localized as separate messages rather than a single message with additional stuff.  (And "was removed" is certainly not localized :D)

Generally speaking, composing messages that would be printed as totally separate lines is OK, but not something you expect to be the same sentence. (Because where you're appending may not be the right place to say that in all languages)